views/index.html

```diff
@@ -2,20 +2,171 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <link rel="icon" type="image/x-icon" href="images/favicon.png">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
+    <link rel="stylesheet" href="style.css">
     <title>Index</title>
 </head>
+
 <body>
-    <header class="bg-dark text-light">
+    <header id="header" class="header">
```

Comment:
is any of this html re-usable and can be moved to a template that will be imported when needed?

Reply:
Yeah just needed to use our new templates from main, sorted now.